functests/1_performance/performance.go

```diff
@@ -781,6 +782,19 @@ func validatTunedActiveProfile(nodes []corev1.Node) {
 	var err error
 	var out []byte
 	activeProfileName := components.GetComponentName(testutils.PerformanceProfileName, components.ProfileNamePerformance)
+
+	// check if some another Tuned profile overwrites PAO profile
+	tunedList := &tunedv1.TunedList{}
+	if err := testclient.Client.List(context.TODO(), tunedList); err != nil {
+		klog.Error(err)
+	}
+	for _, t := range tunedList.Items {
```

Comment:
if you get an error, probably tunedList will be nil

Reply:
error should not happen, added: 	_Expect(err).NotTo(HaveOccurred())_